### ISSUE TITLE
[Feature Request] renamable @module #Heading

### ISSUE THREAD

**Issue by AshleyJamesy** [TurtleP/luaprettydoc#6]
This is just a suggestion, but maybe you could remove the @module keyword.
Lua is flexible and not all lua file will be defined as modules for some developers

Instead @module should be replaced with something to say it's a markdown file, and will create one for this lua file.

Or alternatively you could have a '@header' definition to be able to rename the module header.

Examples:
```
@header Module
@module myclass

# Module myclass
```

I could set it to be:
```
@header Class
@module myclass

# Class myclass
```
```
@header Library 
@module mylibrary

# Library mylibrary
```

**Comment by TurtleP** [comment]
I think that the `@header` tag is probably nicer, realistically. Could rename the module tag to something more generic as well to fit them together.

**Comment by TurtleP** [comment]
Implemented in dd4a8ae.

The data now checks for the following:
```
---[[
@type {Module/Library/???}
@name {what it's called}
@brief {short description--same as before}
--]]
```

**Comment by TurtleP** [comment]
I only thought of validating for `Module` or `Library` as types. Not sure what else people might call something. Comments are welcome regarding this.
